### ISSUE TITLE
instance not found?

### ISSUE THREAD

**Issue by Tealk** [LemmurOrg/lemmur#337]
### Device info

- OS: Android
- OS version: CalyxOS 3.7.0
- Device: Pixel6
- Lemmur version: 0.8.0

### Describe the bug

Cant add my Instance

### Steps to reproduce

1. Go to the Account tab
2. Add account 
3. Add instance
4. Enter https://lemmy.rollenspiel.monster

### Relevant logs

no logs :'(

### Expected behavior

Enter my instance and log in

### Screenshots/Screencasts

![screen](https://user-images.githubusercontent.com/12276250/182219539-7470c428-754a-4d4e-b353-3594157baa47.png)

**Comment by xxxserxxx** [comment]
Same for midwest.social. The domain resolves in the command line, and the server is responding:

```
10009» dig +short midwest.social
206.189.63.152
10010» curl -sI https://midwest.social | grep '^HTTP'
HTTP/2 200
```

What is Lemmur using to test the connection?

**Comment by Occhioverde** [comment]
Possible duplicate of #329; seems fixed in Lemmur 0.9.0

![Working add instance screen](https://user-images.githubusercontent.com/18429412/188514397-90bace2b-ce12-431e-8b93-091b5e0bad10.jpg)

**Comment by Tealk** [comment]
Where i can download 0.9?
